exploits/minitwlpayload/bootloader/source/boot.c

```diff
@@ -66,6 +67,10 @@ extern unsigned long _start;
 extern unsigned long argStart;
 extern unsigned long argSize;
 
+// Manually define as true. DSI exploits always run in DSi mode.
+// Will return as undefined during compile if not defined this way. (with current setup)
+bool dsiMode = true;
```

Comment:
This should be static const, otherwise it's a RW variable that will be put into .data and won't be optimized away